I got this issue when attempting to start the bacnet proxy:
```bash
2025-07-29 12:05:52.972 | DEBUG    | volttron_installer.state:sidebar_selected_page:32 - overview
2025-07-29 12:05:56.026 | DEBUG    | volttron_installer.state:sidebar_selected_page:32 - /tools/bacnet_scan/
2025-07-29 12:05:56.037 | DEBUG    | volttron_installer.state:start_tool:88 - starting tool : bacnet_scan_tool
2025-07-29 12:05:56.037 | DEBUG    | volttron_installer.state:start_tool:99 - setting tool to loading: bacnet_scan_tool
2025-07-29 12:05:56.038 | DEBUG    | volttron_installer.state:start_tool:105 - calling api...
2025-07-29 12:05:56.107 | DEBUG    | volttron_installer.backend.tool_manager:monitor_inactivity:299 - Tool inactivity monitor started (timeout: 30 minutes)
2025-07-29 12:05:56.109 | DEBUG    | volttron_installer.backend.tool_manager:run_service:136 - Starting bacnet_scan_tool with command: /home/os2204/dominicrios-volttron-installer/.venv/bin/python3 -m uvicorn bacnet_scan_tool.main:app --host 0.0.0.0 --port 8001
2025-07-29 12:05:56.110 | DEBUG    | volttron_installer.backend.tool_manager:record_tool_access:44 - Tool access recorded: bacnet_scan_tool at 12:05:56
2025-07-29 12:05:56.924 | DEBUG    | volttron_installer.backend.tool_manager:log_output:161 - [bacnet_scan_tool ERROR] INFO:     Started server process [23426]
2025-07-29 12:05:56.924 | DEBUG    | volttron_installer.backend.tool_manager:log_output:161 - [bacnet_scan_tool ERROR] INFO:     Waiting for application startup.
2025-07-29 12:05:56.928 | DEBUG    | volttron_installer.backend.tool_manager:log_output:161 - [bacnet_scan_tool ERROR] INFO:     Application startup complete.
2025-07-29 12:05:56.929 | DEBUG    | volttron_installer.backend.tool_manager:log_output:161 - [bacnet_scan_tool ERROR] INFO:     Uvicorn running on http://0.0.0.0:8001 (Press CTRL+C to quit)
2025-07-29 12:05:58.111 | DEBUG    | volttron_installer.backend.tool_manager:start_tool_service:207 - Tool processes dictionary keys: ['bacnet_scan_tool']
2025-07-29 12:05:58.111 | DEBUG    | volttron_installer.backend.tool_manager:start_tool_service:208 - Is tool 'bacnet_scan_tool' running? True
2025-07-29 12:05:58.113 | DEBUG    | volttron_installer.state:start_tool:109 - tool started
2025-07-29 12:06:06.001 | DEBUG    | volttron_installer.state:start_proxy:1797 - this is the ip address we will start a proxy with: None
2025-07-29 12:06:06.170 | DEBUG    | volttron_installer.backend.tool_manager:record_tool_access:44 - Tool access recorded: bacnet_scan_tool at 12:06:06
2025-07-29 12:06:09.270 | DEBUG    | volttron_installer.backend.tool_manager:log_output:161 - [bacnet_scan_tool] INFO:     127.0.0.1:56412 - "POST /start_proxy HTTP/1.1" 200 OK
[ERROR] Application callable raised an exception
Traceback (most recent call last):
  File "/home/os2204/dominicrios-volttron-installer/.venv/lib/python3.10/site-packages/granian/_futures.py", line 15, in future_watcher
    await inner(watcher.scope, watcher.proto)
  File "/home/os2204/dominicrios-volttron-installer/.venv/lib/python3.10/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/home/os2204/dominicrios-volttron-installer/.venv/lib/python3.10/site-packages/starlette/middleware/errors.py", line 187, in __call__
    raise exc
  File "/home/os2204/dominicrios-volttron-installer/.venv/lib/python3.10/site-packages/starlette/middleware/errors.py", line 165, in __call__
    await self.app(scope, receive, _send)
  File "/home/os2204/dominicrios-volttron-installer/.venv/lib/python3.10/site-packages/starlette/middleware/cors.py", line 85, in __call__
    await self.app(scope, receive, send)
  File "/home/os2204/dominicrios-volttron-installer/.venv/lib/python3.10/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/home/os2204/dominicrios-volttron-installer/.venv/lib/python3.10/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/home/os2204/dominicrios-volttron-installer/.venv/lib/python3.10/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/home/os2204/dominicrios-volttron-installer/.venv/lib/python3.10/site-packages/starlette/routing.py", line 714, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/home/os2204/dominicrios-volttron-installer/.venv/lib/python3.10/site-packages/starlette/routing.py", line 734, in app
    await route.handle(scope, receive, send)
  File "/home/os2204/dominicrios-volttron-installer/.venv/lib/python3.10/site-packages/starlette/routing.py", line 460, in handle
    await self.app(scope, receive, send)
  File "/home/os2204/dominicrios-volttron-installer/.venv/lib/python3.10/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/home/os2204/dominicrios-volttron-installer/.venv/lib/python3.10/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/home/os2204/dominicrios-volttron-installer/.venv/lib/python3.10/site-packages/starlette/middleware/errors.py", line 187, in __call__
    raise exc
  File "/home/os2204/dominicrios-volttron-installer/.venv/lib/python3.10/site-packages/starlette/middleware/errors.py", line 165, in __call__
    await self.app(scope, receive, _send)
  File "/home/os2204/dominicrios-volttron-installer/.venv/lib/python3.10/site-packages/starlette/middleware/cors.py", line 85, in __call__
    await self.app(scope, receive, send)
  File "/home/os2204/dominicrios-volttron-installer/.venv/lib/python3.10/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/home/os2204/dominicrios-volttron-installer/.venv/lib/python3.10/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/home/os2204/dominicrios-volttron-installer/.venv/lib/python3.10/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/home/os2204/dominicrios-volttron-installer/.venv/lib/python3.10/site-packages/starlette/routing.py", line 714, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/home/os2204/dominicrios-volttron-installer/.venv/lib/python3.10/site-packages/starlette/routing.py", line 734, in app
    await route.handle(scope, receive, send)
  File "/home/os2204/dominicrios-volttron-installer/.venv/lib/python3.10/site-packages/starlette/routing.py", line 288, in handle
    await self.app(scope, receive, send)
  File "/home/os2204/dominicrios-volttron-installer/.venv/lib/python3.10/site-packages/starlette/routing.py", line 76, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/home/os2204/dominicrios-volttron-installer/.venv/lib/python3.10/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/home/os2204/dominicrios-volttron-installer/.venv/lib/python3.10/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/home/os2204/dominicrios-volttron-installer/.venv/lib/python3.10/site-packages/starlette/routing.py", line 73, in app
    response = await f(request)
  File "/home/os2204/dominicrios-volttron-installer/.venv/lib/python3.10/site-packages/fastapi/routing.py", line 327, in app
    content = await serialize_response(
  File "/home/os2204/dominicrios-volttron-installer/.venv/lib/python3.10/site-packages/fastapi/routing.py", line 176, in serialize_response
    raise ResponseValidationError(
ResponseValidationError: 2 validation errors:
  {'type': 'string_type', 'loc': ('response', 'message'), 'msg': 'Input should be a valid string', 'input': None}
  {'type': 'string_type', 'loc': ('response', 'error'), 'msg': 'Input should be a valid string', 'input': None}

```

--- a/volttron_installer/thin_endpoint_wrappers/__init__.py
+++ b/volttron_installer/thin_endpoint_wrappers/__init__.py
@@ -304,7 +304,7 @@ async def create_platform(platform: CreatePlatformRequest):
     await post_request(f"{API_BASE_URL}{PLATFORMS_PREFIX}/", data=platform.model_dump())
 
 async def deploy_platform(platform_id: str, password: str):
-    return await request(f"{API_BASE_URL}{PLATFORMS_PREFIX}/deploy/{platform_id}", "POST", params={"password":password})
+    return await request(f"{API_BASE_URL}{PLATFORMS_PREFIX}/deploy/{platform_id}", "POST", timeout=40.0, params={"password":password})
 
 async def add_host(host: CreateOrUpdateHostEntryRequest):
     await post_request(f"{API_BASE_URL}{HOSTS_PREFIX}", data=host.model_dump())

